debian/argobots-dev.install

```diff
@@ -0,0 +1,8 @@
+# See dh_install(1)
+# http://www.debian.org/doc/manuals/maint-guide/dother.en.html#install
+# run rules/install to find out which file should be copied from debian/tmp
+# to the binary packages.
+# For this package, files in the following directory may be the candidates.
+usr/include
+usr/lib/*/*.so
```

Comment:
Does 64-bit Ubuntu really use `/usr/lib/` for it's libraries?

Reply:
It uses usr/lib/{arch-string}/ for its libraries.  The wildcard is used for a common install.